x-pack/legacy/plugins/infra/public/index.scss

```diff
@@ -18,3 +18,10 @@
   flex: 1 0 0%;
   overflow: hidden;
 }
+
+// This is a temporary workaround for https://github.com/elastic/kibana/issues/39808
+// A real fix will most likely depend on changes in elastic-charts.
+
+.infra-ui-metrics-explorer-chart .echTooltip {
```

Comment:
I just request that you follow the BEM naming structure that's been established. You can find the pattern guidelines at the bottom of this page: https://elastic.github.io/eui/#/guidelines/sass.

Reply:
Thank you for the link. Will change.